lib/video/tasks.py

```diff
@@ -18,8 +20,9 @@
 # Video decoding can take a while, so let's increase these limits.
 @task(time_limit=time_limits['hard'], soft_time_limit=time_limits['soft'])
 @set_modified_on
-def resize_video(src, instance, user=None, **kw):
+def resize_video(src, pk, user=None, **kw):
```

Comment:
that `user` parameter is probably a not going to work either with JSON serialization